sdk/resourcemanager/azure-resourcemanager-test/src/main/java/com/azure/resourcemanager/test/ResourceManagerTestProxyTestBase.java

```diff
@@ -241,7 +241,7 @@ protected void beforeTest() {
         if (isPlaybackMode()) {
             if (interceptorManager.getRecordedData() == null) {
                 skipInPlayback();
-                return;
+//                return;
```

Comment:
Remove `return` here because when using testproxy, `interceptorManager.getRecordedData()` will be `null`, when running in playback mode, below logics to initialize clients will be skiped.

Reply:
Since this is TextProxyTestBase, I take `interceptorManager.getRecordedData()` is always `null`, and `skipInPlayback()` is always called?

And why do we want `skipInPlayback()` always called? (regardless whether the test want to skip in playback or not)

Reply:
`isSkipInPlayback = true` seems meaning it does not clean up the resource after test complete?

Reply:
> `isSkipInPlayback = true` seems meaning it does not clean up the resource after test complete?

Yes, if we set to true, it will not do test resource clean up calls. Previously I think it is playback mode, there is no need to do resource clean up. But to be consistent with previous logic, we can do resouce clean up as well, that is to delete below whole part. What do you think?
```java
 if (isPlaybackMode()) {
            if (interceptorManager.getRecordedData() == null) {
                skipInPlayback();
                return;
             }
}
```

Reply:
Yeah, you can do this.

Later we may get to test case with `@DoNotRecord(skipInPlayback = true)`, which may have problem (I kind of assume code here was to handle that, but not sure). You can handle that later.

Reply:
`@DoNotRecord(skipInPlayback = true)` -> there is this case in resources. I will try on playback and record mode.